src/wallet/wallet.h

```diff
@@ -74,13 +74,13 @@ std::unique_ptr<WalletDatabase> MakeWalletDatabase(const std::string& name, cons
 //! -paytxfee default
 constexpr CAmount DEFAULT_PAY_TX_FEE = 0;
 //! -fallbackfee default
-static const CAmount DEFAULT_FALLBACK_FEE = 0;
+static const CAmount DEFAULT_FALLBACK_FEE = 100000;
 //! -discardfee default
 static const CAmount DEFAULT_DISCARD_FEE = 10000;
 //! -mintxfee default
-static const CAmount DEFAULT_TRANSACTION_MINFEE = 1000;
+static const CAmount DEFAULT_TRANSACTION_MINFEE = 100000;
 //! -consolidatefeerate default
-static const CAmount DEFAULT_CONSOLIDATE_FEERATE{10000}; // 10 sat/vbyte
+static const CAmount DEFAULT_CONSOLIDATE_FEERATE{1000000}; // 1000 sat/vbyte
```

Comment:
Since our minimum feerate is 100 sat/vbyte (100x greater than bitcoin), consolidatefeerate has to be set to a higher threshold for the logic to work so I increased it by the same factor.
This was implemented in https://github.com/bitcoin/bitcoin/pull/22009

From a comment in the code: "When the actual feerate is less than the consolidate feerate, we will tend to make transactions which consolidate inputs. When the actual feerate is greater than the consolidate feerate, we will tend to make transactions which have the lowest fees." 
https://github.com/bitcoin/bitcoin/pull/22009/files#diff-9ce137cd784ea308778842120aa2af6d2bb8369485b71f25e72b2a32cf0a5b21R643